src/Azure.Functions.Cli/Actions/HostActions/StartHostAction.cs

```diff
@@ -727,5 +728,24 @@ internal static bool ConnectionExists(IEnumerable<KeyValuePair<string, string>>
                 : null;
             return (new Uri($"{protocol}://0.0.0.0:{Port}"), new Uri($"{protocol}://localhost:{Port}"), cert);
         }
+        private void CheckSettingFunctionWorkerRuntime()
+        {
+            var setting = _secretsManager.GetSecrets().FirstOrDefault(s => s.Key.Equals(Constants.FunctionsWorkerRuntime, StringComparison.OrdinalIgnoreCase)).Value;
```

Comment:
Can you be more specific about the setting name? i.e. `workerRuntimeSettingValue`